### PR TITLE
plat-stm: fix traces

### PR DESCRIPTION
**Pull request by pascal-brand-st-dev**
On plat-stm, traces IP is initialized by Non Secure world.
Hence no traces can be output while in the booting process.

Signed-off-by: Pascal Brand <pascal.brand@st.com>

### CODE REVIEW

File: core/arch/arm/plat-stm/main.c [hunk 1]
```diff
@@ -37,9 +37,10 @@
 #include <asc.h>
 
 static void main_fiq(void);
+static void my_tee_entry_std(struct thread_smc_args *smc_args);
 
 static const struct thread_handlers handlers = {
-	.std_smc = tee_entry_std,
+	.std_smc = my_tee_entry_std,
```
Comment (jenswi-linaro) [inline core/arch/arm/plat-stm/main.c]: I would prefer `stm_tee_entry_std`.

File: core/arch/arm/plat-stm/tz_a9init.S [hunk 2]
```diff
@@ -47,37 +44,6 @@
 .code 32
 
 /*
- * arm_secboot_identify_cpu - identify and save CPU version
```
Comment (jenswi-linaro) [inline core/arch/arm/plat-stm/tz_a9init.S]: This seems to be unrelated to traces. Update commit message or split into two?

**Comment by pascal-brand-st-dev** [comment]
updated

**Comment by jenswi-linaro** [comment]
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`
